FirebaseCrashlytics.podspec

```diff
@@ -45,8 +45,8 @@ Pod::Spec.new do |s|
   s.dependency 'FirebaseInstallations', '~> 1.1'
   s.dependency 'FirebaseAnalyticsInterop', '~> 1.2'
   s.dependency 'PromisesObjC', '~> 1.2'
-  s.dependency 'GoogleDataTransport', '~> 4.0'
-  s.dependency 'GoogleDataTransportCCTSupport', '~> 1.3'
+  s.dependency 'GoogleDataTransport', '~> 4.0', '>= 4.0.1'
```

Comment:
If you can hold off on submitting this, probably pin to the soon-to-be-released GDT 5.0 and GDTCCT 2.0 versions.

Reply:
Ideally internal testing should depend on GitHub latest.

Reply:
Since Firebase by default enables minor version updates of other pods in the repo - if a major version update is made by a low level pod, that pod should probably also manage the migration of its dependents so that the repo stays consistent at latest versions. Does that work for you @mikehaney24? cc: @ryanwilson @maksymmalyhin

Reply:
I don't think it's reasonable in the general case for a low-level library to manage the version dependencies of its adopters (what if they pinned to a lower version on purpose?) In this case, I can take care of it going forward, but only for Firebase OSS SDKs.

Reply:
Sorry, I should have been clear that I just meant dependencies in this repo.  Does that work?

Reply:
Yeah, maybe there's a way to make sure we always change it Firebase-wide by having a ruby config file with GDT versions that's loaded by the podspecs? For now, I can do a find-and-replace after major version changes.

Reply:
For this change we'll hold off for V5 / V2